tests/performance_estimation/test_cbpe.py

```diff
@@ -242,3 +242,25 @@ def test_cbpe_for_binary_classification_does_not_fail_when_fitting_with_subset_o
             'fitting on subset resulted in KeyError => misaligned indices between data and stratified shuffle'
             'split results.'
         )
+
+
+def test_cbpe_for_binary_classification_does_not_output_confidence_bounds_outside_appropriate_interval(
+    monkeypatch, metadata, data
+):
+    reference, analysis = data
+    estimator = CBPE(model_metadata=metadata, metrics=['roc_auc'])
+    estimator.fit(reference_data=reference)
+    results = estimator.estimate(pd.concat([reference, analysis]))
+    min_confidence = results.data['lower_confidence_roc_auc'].min()
+    max_confidence = results.data['upper_confidence_roc_auc'].max()
+
+    # When changing the boundaries to values inside the current range of values
+    new_lower_bound = min_confidence + 0.001
+    new_upper_bound = max_confidence - 0.001
+    monkeypatch.setattr(estimator, 'metric_lower_bound', new_lower_bound)
```

Comment:
An alternative could be adding optional arguments to the `_BinaryClassificationCBPE.init` function that set the values of the boundaries and default them to 0 and 1.  

This would eliminate the need for monkeypatching, but I'm not sure if there is a lot of added value about exposing the upper/lower confidence boundaries.

So for me the implementation as is is fine.

Reply:
Ok, I would rather keep it that way then.